docs/settings/security-settings.asciidoc

```diff
@@ -337,24 +337,12 @@ For more details and a reference of audit events, refer to <<xpack-security-audi
 [cols="2*<"]
 |======
 | `xpack.security.audit.enabled` {ess-icon}
-| Set to `true` to enable audit logging for security events. *Default:* `false`
-|======
-
-[float]
-[[ecs-audit-logging-settings]]
-==== ECS audit logging settings
-
-To enable the <<xpack-security-ecs-audit-logging, ECS audit logger>>, specify where you want to write the audit events using `xpack.security.audit.appender`.
-
-[cols="2*<,*50"]
-|======
-| `xpack.security.audit.appender`
-| Optional. Specifies where audit logs should be written to and how they should be formatted.
+| Set to `true` to enable audit logging and configure where the audit log should be written to using `xpack.security.audit.appender` . *Default:* `false`
```

Comment:
Looking great!

One last thing:

Re-reading this, from the "Audit logs" page it implies that you need `xpack.security.audit.appender` to use ECS audit events: https://kibana_104685.docs-preview.app.elstc.co/guide/en/kibana/master/xpack-security-audit-logging.html#xpack-security-ecs-audit-logging

However, that links to the Audit logging settings page: https://kibana_104685.docs-preview.app.elstc.co/guide/en/kibana/master/security-settings-kb.html#audit-logging-settings
And that page actually doesn't mention that it enables ECS audit logging. Further down, it _does_ mention what you need to do to enable only _legacy audit logging_, though.

What do you think about changing this to make it clear?

```suggestion
| Set to `true` _and_ configure an appender with `xpack.security.audit.appender` to enable ECS audit logging` . *Default:* `false`
```